src/main/java/com/flightstats/hub/spoke/SpokeTtlEnforcer.java

```diff
@@ -26,37 +23,33 @@
 @Singleton
 @Slf4j
 public class SpokeTtlEnforcer {
-    private final SpokeStore spokeStore;
+
     private final ChannelService channelService;
     private final SpokeContentDao spokeContentDao;
     private final StatsdReporter statsdReporter;
+    private final SpokeProperties spokeProperties;
 
-    private final String storagePath;
-    private final int ttlMinutes;
+    private SpokeStore spokeStore;
+    private String storagePath;
+    private int ttlMinutes;
 
     @Inject
-    public SpokeTtlEnforcer(SpokeStore spokeStore,
-                            ChannelService channelService,
+    public SpokeTtlEnforcer(ChannelService channelService,
                             SpokeContentDao spokeContentDao,
                             StatsdReporter statsdReporter,
                             SpokeProperties spokeProperties) {
-        this.spokeStore = spokeStore;
         this.channelService = channelService;
         this.spokeContentDao = spokeContentDao;
         this.statsdReporter = statsdReporter;
-        this.storagePath = spokeProperties.getPath(spokeStore);
-        this.ttlMinutes = spokeProperties.getTtlMinutes(spokeStore) + 1;
-        if (spokeProperties.isTtlEnforced()) {
-            HubServices.register(new SpokeTtlEnforcerService());
-        }
+        this.spokeProperties = spokeProperties;
     }
```

Comment:
Happy to see this service-registration-in-constructors model getting killed off.  :)